resources/hapi_page_url_allowed_queries.json

```diff
@@ -9,19 +9,19 @@
       "allParamsRequired": true
     },
     {
-      "path": "Composition/ANY_VALUE",
+      "path": "Composition",
       "methodType": "GET",
       "queryParams": {
-
+        "identifier":"ANY_VALUE"
       },
       "allowExtraParams": true,
-      "allParamsRequired": false
+      "allParamsRequired": true
     },
     {
-      "path": "Binary/ANY_VALUE",
+      "path": "Binary",
       "methodType": "GET",
       "queryParams": {
-
+        "_id":"ANY_VALUE"
       },
       "allowExtraParams": true,
       "allParamsRequired": false
```

Comment:
We need to update the `allParamsRequired` to true to enforce that an `_id` param is required for the` /Binary` endpoint

Reply:
I see what you mean.